src/description/eth_blockNumber.md

```diff
@@ -0,0 +1 @@
+* If the client is synced to any network it **MUST** return the current finalized block of that network, otherwise it **MUST** return `0x0`
```

Comment:
```suggestion
 * The client **MUST** return the latest fully validated/verified block on the best known canonical chain.
```

Finalization has a very specific meaning in proof of stake, and none in proof of work and shouldn't be used here.

I changed the wording to reflect what should (and I believe is) be happening here, though I think even my wording may be a bit too strong because it doesn't include light clients which don't fully validate/verify blocks.

Reply:
Wouldn't we also want to describe the else case?
Where the client cannot return the block or do we just want to describe the ideal expected behavior?

Reply:
@MicahZoltu

Reply:
There is no else case, returning `0` falls into the primary case.  :)  If you haven't synced *any* blocks then you *at least* have genesis.  However, in the future we may have things like regenesis, where we sync from not-0, in which case you would want to return the regenesis block which would not be `0`.

Reply:
Also, once someone comments on a PR they are automatically subscribed, so no need to mention.  :)

Reply:
I see what you mean that makes sense to me. 
@MicahZoltu I will not mention you in PRs you are already subscribed to.....  After this one :)